lib/c-api/tests/wasmer-c-api-test-runner/src/lib.rs

```diff
@@ -220,7 +220,7 @@ fn test_ok() {
             println!("setting current dir = {exe_dir}");
             let output = command
                 .output()
-                .expect(&format!("failed to run {command:#?}"));
+                .unwrap_or_else(|_| panic!("failed to run {command:#?}"));
```

Comment:
this is a clippy autofix done by clippy --fix --all, it does exactly the same thing, but it doesn't construct a string in the "okay" case. wontfix because it will be reapplied by `clippy --fix` on the next run anyways.